docs/introduction/Car-registry-tutorial.rst

```diff
@@ -2,73 +2,76 @@
+Accepting that cars will show up in the sidechain in our example, we want to build an application that has the following capabilitis:
 
+    1. It can store information that identifies a specific car, such as vehicle identification number (VIN), model, production year, color.
+    2. Allows car owners to be able to prove their ownership of the cars anonymously.
+    3. Require the use of ZEN for all transactions. 
+
+(Not sure that "Sidechain" should be capitalized as often as it is, but left it in case it's an internal formatting choice.)
 
 User stories:
 #############
 
 1
-**Q: I want to add my car to a Car Registry Sidechain.**
+**Q: I want to add my car to the Car Registry Sidechain.**
 
-*A:* Create a new Car Entry Box, which contains car identification information (Unique car identifier, VIN, manufacturer, model, year, registration number), and certificate. Proposition in this box is my public key in this Sidechain. When I create a box, Sidechain should check car identification information and certificate to be unique in this Sidechain.
+*A:* Create a new Car Entry Box, which contains car identification information (VIN, manufacturer, model, year, registration number), and certificate. Proposition in this box is my public key in this Sidechain. When I create a box, the Sidechain should verify car identification information and certificate are unique in this Sidechain.
 
 2
-**Q: I want to create sell order to sell my car using Car Registry Sidechain.**
+**Q: I want to create a sell order to sell my car using Car Registry Sidechain.**
 
-*A:* I create a new Car Sell Order Box that contains the price in coins and information from the Car Entry Box. So cars can exist in the Sidechain as a Car Entry Box or as a Car Sell Order, but not at the same time. Also, this box contains the buyer’s public key. When I create a sell order, Sidechain should check if there is no other active sell order with this Car Entry Box. The current Sell Order consists of the same information that consists of the Car Entry Box plus description.
+*A:* I create a new Car Sell Order Box that contains the price in coins and information from the Car Entry Box. So cars can exist in the Sidechain either as a Car Entry Box or as a Car Sell Order, but not both at the same time. Also, this box contains the buyer’s public key. When I create a sell order, the Sidechain should verify that there is no other active sell order with this Car Entry Box. The current Sell Order consists of the same information that contained in the Car Entry Box plus description (is the description optional?).
 
 3
-**Q: I want to see all available Sell orders in Sidechain**
+**Q: I want to see all available Sell orders in the Sidechain.**
 
 *A:* Have additional storage, which is managed by ApplicationState and stores all Car Sell Orders. All these orders can be retrieved using the new HTTP API call. 
```

Comment:
*A:* Have additional storage, which is managed by ApplicationState and stores all Car Sell Orders. All orders can be retrieved using the new HTTP API call. 

@zennermac - What has additional storrage?